### DISCUSSION TITLE
How to make the second TabPane be default opening view?

### DISCUSSION THREAD

**Discussion by MohamedElgamal** [Textualize/textual#3234] [Q&A] [answered]
Hello Everyone, I have application that consist of three Tab Pane, I want to make the second tab be the default tab to appear.

**Comment by TomJGooding** [comment, marked as answer]
I would really recommend taking some time to read the Textual documentation. Here's the link again to the `TabbedContent` docs: https://textual.textualize.io/widgets/tabbed_content 

Hint: You probably want to check how to set the initial tab.

**Reply by MohamedElgamal** [reply]
thanks, is active method do the same if yes can you provide simple example? and what is the difference between show_tab and active method?

**Reply by davep** [reply]
[`active` is a reactive](https://textual.textualize.io/widgets/tabbed_content/#textual.widgets._tabbed_content.TabbedContent.active) -- if you assign an ID to it then the tab pane with that ID will become the active tab. You can see an example of this [in the documentation that Tom recommended to you](https://textual.textualize.io/widgets/tabbed_content/#__tabbed_1_2).
